warehouse/warehouse.go

```diff
@@ -87,6 +83,15 @@ const (
 	EmbeddedMode    = "embedded"
 )
 
+// warehouse table names
+const (
+	warehouseStagingFilesTable = "wh_staging_files"
+	warehouseLoadFilesTable    = "wh_load_files"
+	warehouseUploadsTable      = "wh_uploads"
+	warehouseTableUploadsTable = "wh_table_uploads"
+	warehouseSchemasTable      = "wh_schemas"
+)
+
```

Comment:
let's remove this in config.toml also and use from constants in other files like warehouseutils also?

Reply:
Yeah. we should do this.